scripts/decode-eth-eip1559.js

```diff
@@ -0,0 +1,16 @@
+const { Transaction: TransactionFactory, FeeMarketEIP1559Transaction } = require('@ethereumjs/tx');
+const ethUtil = require('ethereumjs-util');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+main();
+
+function main() {
+  readline.question('Enter the transaction hex from your recovery JSON: ', (data) => {
```

Comment:
who is the customer for this script? also rather than adding `0x` unconditionally you can use `startsWith` to see if it already exists, and only add it if it doesn't.